recon_surf/recon-surf.sh

```diff
@@ -252,6 +261,23 @@ then
   export PYTHONUNBUFFERED=0
 fi
 
+basedir=""
+if [ "$long" == "1" ]
+then
+  basedir="$SUBJECTS_DIR/$baseid"
+  if [ ! -f "$basedir/base-tps.fastsurfer" ] ; then
+    echo "$baseid is either not found in SUBJECTS_DIR or it is not a longitudinal template"
+    echo "directory, which needs to contain base-tps.fastsurfer file. Please ensure that"
+    echo "the base (template) has been created when running with --long flag."
+    exit 1
+  fi
+  if [ ! grep -Fxq $basedir/base-tps.fastsurfer $subject ] ; then
+    echo "$subject id not found in base-tps.fastsurfer. Please ensure that this time point"
+    echo "was included during creation of the base (template)."
+    exit 1
+  fi
+fi
+
```

Comment:
We also probably need some kind of check to see if this is both a long and a base run (which seems to be invalid)